int8 type
int8 number
int8 value
